pipeline/azure-pipelines.yml

```
@@ -17,7 +17,25 @@ variables:
   isMaster: $[eq(variables['Build.SourceBranch'], 'refs/heads/master')]
   connectedServiceName: DCD-CFT-Sandbox
   keyVaultName: infra-vault-sandbox
-  secretsFilter: 'azure-devops-token'
+  secretsFilter: azure-devops-token
+  project: application
+
+parameters:
```

can this be a variable we shouldn't have this showing up in the UI as a parameter

---

sorry, i didn't catch your comments. What are we referring to here ?

---

something like this:
```suggestion
variables:
```

parameters show up in the UI when you click run new